docker-process.el

```diff
@@ -26,16 +26,22 @@
 (require 's)
 (require 'dash)
 
+(defcustom docker-use-root-privilege nil
+  "Boolean value indicating if the docker command will run under root privilege."
```

Comment:
`docker-run-as-root` is a better name no?

Also, the docstring should be `Controls wether docker runs as root or not.`

Reply:
Certainly :). I am not very good at naming variable and writing docstring.